### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by ShaliniNayan**
Hello,

For this PR, I have a Vet clinic database and added more tables and update animals tables for the project using the:

- Create two new tables named owners and species with columns (id: integer (set it as autoincremented PRIMARY KEY), 
   full_name: string, age: integer & id: integer (set it as autoincremented PRIMARY KEY), name: string, ).

- Modify the animals table:
      - Id is set as autoincremented PRIMARY KEY.
      - Remove column species.
      - Add column species_id which is a foreign key referencing species table.
      - Add column owner_id which is a foreign key referencing the owners table.
- The SQL used for creating database and table is written in schema.sql
- Data inserted into the owners table like name and age.
- Data inserted into species table like the name.
- Modify inserted animals it includes the species_id value.
- Modify inserted animals to include owner information (owner_id)
- The SQL used for modifying data in the table is written in data.sql.
- The SQL for Queries using (JOIN) is written queries.sql and screenshots of the results of your queries are as follows :
image
image
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,3 +11,31 @@ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
 insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Ditto', '2022-05-14', 4, true, 22);
+
+-- Data Entries for Owners
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
```
Comment (Ismailco) [inline data.sql:22]: - [x] Kindly make sure to add the required data to the species table e.g 
```sql
 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');
```

File: data.sql [hunk 2]
```diff
@@ -11,3 +11,31 @@ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+-- Update Data Animals
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE species_id IS NULL;
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') WHERE name = 'Agumon';
+
+UPDATE animals
+vet_clinic-# SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') WHERE name IN ('Gabumon', 'Pikachu');
```
Comment (Ismailco) [inline data.sql]: - [x] I think you commented this line accidentally, kindly make sure to remove `vet_clinic-# ` so the code works normally

Comment (Ismailco) [inline data.sql]: - [x] I think you commented this line accidentally, kindly make sure to remove `vet_clinic-# ` so the code works normally

File: queries.sql [hunk 3]
```diff
@@ -31,3 +31,11 @@ select avg(weight_kg) from animals;
 select neutered, max(escape_attempts) as max_escape_attempts from animals group by neutered;
 select species, avg(escape_attempts) as avg_escape_attempts from animals where date_of_birth between '1990-01-01' and '2000-12-31' group by species;
 select species, min(weight_kg) AS min_weight, max(weight_kg) as max_weight from animals group by species;
+
+SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id WHERE o.full_name = 'Melody Pond';
+SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id WHERE s.name = 'Pokemon';
```
Comment (Ismailco) [inline queries.sql]: - [x] In these two queries you should select all the data not only names, kindly consider using the `*` to select all the data instead

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ShaliniNayan, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (ShaliniNayan) [reply queries.sql]: image

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @ShaliniNayan 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done updating the data file according to the requested changes from the previous reviewer

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**